rosidl_typesupport_opensplice_c/resource/msg__type_support_c.cpp.em

```diff
@@ -61,11 +61,6 @@ extern "C"
 {
 #endif
 
-// Forward declare the get type support function for this type.
-ROSIDL_GENERATOR_C_EXPORT_@(spec.base_type.pkg_name)
-const rosidl_message_type_support_t *
-  ROSIDL_GET_TYPE_SUPPORT_FUNCTION(@(pkg), @(subfolder), @(msg))();
-
```

Comment:
Unfortunately that only defines the `ROSIDL_GET_TYPE_SUPPORT_FUNCTION` macro, but does not use it to declare the message specific version of the function. That is done here:

https://github.com/ros2/rosidl/blob/d09b0edf92d3bc5cf05974fbff20a78ee1ddf70a/rosidl_generator_c/resource/msg__type_support.h.em#L46-L49